schedule/yast/skip_registration/skip_registration@svirt-hyperv.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name:           skip_registration@svirt-hyperv
+description:    >
+  Like a standard scenario with explicit skipping of SCC registration
+  in case where we register by default, e.g. for SLE >= 15
+  See https://progress.opensuse.org/issues/25264 for details.
+
+schedule:
+  #- installation/bootloader_hyperv
```

Comment:
I guess the commented line is not needed anymore. Could you please remove commented line from the commit.

Reply:
sure thing.